README.md

```diff
@@ -53,6 +53,11 @@ kubectl create -f csi-nodeplugin-rbac.yaml
 kubectl create -f csi-nodeplugin-do.yaml
 ```
 
+A new storage class will be created with the name `do-block-storage` which is
```

Comment:
`This is set to default` -> `This is set by default` or `This is true by default`?

Reply:
If you install a storage class, it's not set to `default` setting. If you mean `set by default`, the question boils down on `what` is set to default. If I change then it should be something like `Thi is set by default to "default"` which is not what we want?

Reply:
Changed it to `This is set to "default" for dynamic provisioning.`